flink-cyber/metron-parser-chain/parser-chains-parsers/src/test/java/com/cloudera/parserchains/parsers/StellarParserTest.java

```diff
@@ -109,7 +111,7 @@ private Message verifyParser(String inputField, String configPath) throws IOExce
         return output;
     }
 
-    private static File getFileFromResource(String path) {
+    static File getFileFromResource(String path) {
```

Comment:
why you changed this method signature?

Reply:
To reuse the code in the test for my class. I'll extract it into shared part